polaris-react/.storybook/preview-head.html

```diff
@@ -6,6 +6,6 @@
 />
 <link
   id="inter-font-link"
-  href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap"
+  href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;450;500;600;650;700;800;900&display=swap"
```

Comment:
I saw you removed 100, 200, 300, 800, 900 in other places, can they be removed here too?

Reply:
I'll keep the 400, 500, 600 in just in case we have 3P developers that are still using those in their apps, but we should remove them later.